src/pages/ModelizerPage.vue

```diff
@@ -13,7 +13,10 @@
         class="modelizer-page column"
         data-cy="modelizer-page"
       >
-        <modelizer-model-view v-show="viewType === 'model'" />
+        <modelizer-model-view
+          v-show="viewType === 'model'"
+          :project-name="projectName"
+        />
         <modelizer-text-view
           v-show="viewType === 'text'"
           :viewType="viewType"
```

Comment:
to delete `:viewType="viewType"` is props not used inside ModelizerTextView

Reply:
Done